### PR TITLE
Notebooks

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/notebooks/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Notebook
+nav_order: 38
+has_children: false
+---
+
+# Kibana Notebook
+
+A Kibana notebook is an interface that lets you easily create and share live log data reports.
```
Comment (anirudha) [inline docs/notebooks/index.md:10]: one of the key differentiators is 
- ability to view multi-timeline reports

File: docs/notebooks/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (anirudha) [inline docs/notebooks/index.md:1]: the gif above is very old version of the software, where did you get it from ?

File: docs/notebooks/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+You can perform the following actions on paragraphs:
+
+- Add a new paragraph to the top of a report.
+- Add a new paragraph to the bottom of a report.
+- Run all the paragraphs at the same time.
+- Clear the outputs of all paragraphs.
+- Delete all the paragraphs.
+
+## Import Sample Notebooks
```
Comment (anirudha) [inline docs/notebooks/index.md:57]: import is not supported now.

File: docs/notebooks/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+To run and save a paragraph, choose **Run**.
+
+You can perform the following actions on paragraphs:
+
+- Add a new paragraph to the top of a report.
+- Add a new paragraph to the bottom of a report.
+- Run all the paragraphs at the same time.
+- Clear the outputs of all paragraphs.
+- Delete all the paragraphs.
+
```
Comment (anirudha) [inline docs/notebooks/index.md:56]: there are other options to move paragraphs up/down too.

File: docs/notebooks/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+## Get Started with Notebooks
+
+To get started, choose **Kibana Notebooks** in Kibana.
+
+### Step 1: Create a notebook
+
+A notebook is an interface for creating reports.
+
+1. Choose **Create notebook**.
+- In the pop-up, enter a descriptive name.
```
Comment (aetter) [inline docs/notebooks/index.md:26]: Just add this to first line. Choose **Create notebook** and enter a descriptive name.

File: docs/notebooks/index.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+1. Choose **Create**.
+
+Choose **Notebook actions** to rename, duplicate, or delete a notebook.
+
+### Step 2: Add a paragraph
+
+Paragraphs combine text and visualizations for describing data.
+
+#### Add a markdown paragraph
```
Comment (aetter) [inline docs/notebooks/index.md:35]: Capitalize Markdown and add a link to documentation for whatever flavor we're using (probably CommonMark or GFM) in the description.

Examples:

https://github.github.com/gfm/
https://commonmark.org

File: docs/notebooks/index.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+#### Add a visualization paragraph
+
+1. To add a visualization, choose **Add Kibana visualization paragraph**.
+1. In **Title**, select your visualization and choose a date range.
+- You can choose multiple timelines to compare and contrast visualizations.
+1. To resize a visualization, unpin the visualization and enter the edit mode. After you resize the visualization, choose **Save**.
+
+To run and save a paragraph, choose **Run**.
+
+You can perform the following actions on paragraphs:
```
Comment (aetter) [inline docs/notebooks/index.md:49]: Is there no sequential drilling down on data as you progress through the notebook? It's just visualizations and paragraphs?

Comment (vrphanse) [reply docs/notebooks/index.md:49]: When the inspect button on output only is clicked, it shows the data. @anirudha to confirm.

File: docs/notebooks/index.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/notebooks/index.md:10]: Does multi-timeline mean adding two visualizations paragraphs with different date ranges for comparative analysis?

Comment (vrphanse) [reply docs/notebooks/index.md:10]: Please refer to What's new

**Pull request merged**